### DISCUSSION TITLE
Where could I find the `secret_prov_gramine.pc` ?

### DISCUSSION THREAD

**Discussion by blank-vax** [gramineproject/gramine#1296] [General]
I am trying to execute the examples in CI-Example directory in the gramineproject/gramine docker. Unfortunately, some errors appear when I perform the following instruction to build the files in the ra-tls-secret-prov directory.
The instruction is as that introduced in the CI-Example/ra-tls-secret-prov/README.md:

```
make app epid RA_TYPE=epid RA_CLIENT_SPID=my_intel_SPID RA_CLIENT_LINKABLE=0
```
Here are my error information:
```
Package secret_prov_gramine was not found in the pkg-config search path/
Perhaps you should add the directory containing 'secret_prov_gramine.pc' to the PKG_CONFIG_PATH environment variable
No package 'secret_prov_gramine' found
```
Finally, I got the error like this:
```
secret_prov/client.c:12:10: fatal error: secret_prov.h: No such file or directory
```
Could anyone assist me?

**Comment by dimakuv** [comment]
@blank-vax The Docker image has Gramine v1.4. So you should use the corresponding tag in the cloned Gramine repo:
```bash
git clone https://github.com/gramineproject/gramine.git
cd gramine

git checkout v1.4  # this is important!
cd CI-Examples/ra-tls-secret-prov
...
```

**Reply by blank-vax** [reply]
That does work! Thank u so much!
